### DISCUSSION TITLE
Dragging windows sometimes doesn't work - Win x64

### DISCUSSION THREAD

**Discussion by beerman-uk** [f1multiviewer/issue-tracker#27] [General]
Picking up and dragging the different windows around is quite hard.  Could it be enabled so any where you click in the window allows you to drag it?  I found 50% of the time clicking the top and dragging did nothing.  I had to be quite precise when picking up a window.

**Comment by loheiman** [comment]
Same issue here. It appears to require clicking once to bring up the title bar then dragging. But even when doing that it often doesn't work for some reason.

**Comment by jeppeolesen** [comment]
Hey there. If this is still relevant, can you create it as a bug under issues: https://github.com/f1multiviewer/issue-tracker/issues
